doc/nrf/protocols/matter/getting_started/low_power_configuration.rst

```diff
@@ -153,6 +154,18 @@ Switching the Matter :ref:`ug_matter_device_low_power_icd_modes` and frequently
 To avoid this issue, set the :kconfig:option:`CONFIG_CHIP_ICD_SLOW_POLL_INTERVAL` and :kconfig:option:`CONFIG_CHIP_ICD_FAST_POLLING_INTERVAL` Kconfig options to the same value (for example, ``500``).
 The typical use case that the SSED is best suited for is battery-powered devices that require short response time, such as door locks or window blinds.
 
+Child timeouts configuration
+----------------------------
+
+The device working in a Thread child role uses additional mechanisms for periodically ensuring that the communication with the parent is still possible.
+These mechanisms lead to waking up the device and exchanging the messages with the parent, if the related timeout expires.
+In case of using the SED poll period value greater than any of these timeouts, the device is going to wake up more often, than it is defined by the poll period.
```

Comment:
```suggestion
In case of using the SED poll period value greater than any of these timeouts, the device wakes up more often than what is defined by the poll period.
```